Arelle/Arelle
# Fix Linux Build

#### Reason for change
The pillow directory copied as the part of the Linux build changed from `Pillow.libs` to `pillow.libs` [in the new 10.2.0 pillow release](https://github.com/Arelle/Arelle/pull/1025) breaking the Linux build.

#### Description of change
* Make Python lib copy script case insensitive.
* Update Ubuntu image used for Linux build to oldest Ubuntu LTS release.

#### Steps to Test
* Verify [Linux build for this branch](https://github.com/austinmatherne-wk/Arelle/actions/runs/7415935629) completes.

**review**:
@Arelle/arelle
